lib/propcheck.ex

```diff
@@ -663,6 +663,20 @@ defmodule PropCheck do
         ...>   end)
         true
 
+    You also can refer variables declared in var scope with `^` until they
+    not form cycle or mix with var with same name from outer scope.
```

Comment:
I would go for this: 

You als can refer *to* variables declared in *the current `let`* scope with `^`. Such variables are not allowed to form a dependency cycle . You cannot apply `^` to variables declared outside of the `let` macro.

Reply:
Thank you >_>